Source/WebKit/UIProcess/PageLoadState.h

```diff
@@ -269,4 +269,65 @@ class PageLoadState {
     unsigned m_outstandingTransactionCount;
 };
 
+inline bool PageLoadState::isLoading() const
+{
+    return isLoading(m_committedState);
+}
+
+inline bool PageLoadState::hasUncommittedLoad() const
+{
+    return isLoading(m_uncommittedState);
+}
+
+inline bool PageLoadState::isLoading(const Data& data)
```

Comment:
I'll let @rniwa and @geoffreygaren comment but it doesn't seem *that* small, not sure we want to inline this.

Reply:
I agree. I think it's already trivial access without inlining and I hope analyzer can detect this well.

Reply:
To clarify, the analyzer can only detect that a function is trivial if the function is inline. So, we have to choose between
* Make a function inline, so we can avoid RefPtr
* Use RefPtr, so we can keep a function out-of-line

Given that this inline doesn't require new #includes, maybe inline is OK?

Note that 'inline' does not mandate emitting the body of the function into the caller, and non-'inline' does not mandate the opposite, either. The compiler uses its own judgment. It can out-of-line a function marked 'inline', or inline a function not marked 'inline' (if its definition is visible in the translation unit, which it often is in our unified builds).

Reply:
> Given that this inline doesn't require new #includes, maybe inline is OK?

If it doesn't bloat the binary / hurt perf.